bso/structural_design/element/element.cpp

```diff
@@ -136,6 +136,11 @@ namespace bso { namespace structural_design { namespace element {
 		return this->getVolume();
 	} // getVolumeSensitivity()
 
+	double element::getStressCenter(const double& alpha /* 0*/, const double& beta /* 1.0 / sqrt(3)*/) const
+	{
+		throw std::runtime_error("cannot call getStressCenter() for this element type");
```

Comment:
Please also mention the location within the code where this exception is thrown. For this, see how exceptions are thrown in other parts of the code.